plugins/scaffolder-backend/src/scaffolder/actions/builtin/github/OctokitProvider.ts

```diff
@@ -35,18 +34,14 @@ export type OctokitIntegration = {
  */
 export class OctokitProvider {
   private readonly integrations: ScmIntegrationRegistry;
-  private readonly credentialsProviders: Map<string, GithubCredentialsProvider>;
+  private readonly githubCredentialsProvider: GithubCredentialsProvider;
 
-  constructor(integrations: ScmIntegrationRegistry) {
+  constructor(
+    integrations: ScmIntegrationRegistry,
+    githubCredentialsProvider: GithubCredentialsProvider,
+  ) {
     this.integrations = integrations;
-    this.credentialsProviders = new Map(
-      integrations.github.list().map(integration => {
-        const provider = SingleInstanceGithubCredentialsProvider.create(
-          integration.config,
-        );
-        return [integration.config.host, provider];
-      }),
-    );
+    this.githubCredentialsProvider = githubCredentialsProvider;
```

Comment:
One might recognize the code that was replaced here x)

Reply:
:D